pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/AdditionalOptions/AdditionalOptions.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react';
+import { CheckboxField } from '@percona/platform-core';
+import { Field } from 'react-final-form';
+import { RadioButtonGroupAdapter } from 'shared/components/Form/FieldAdapters/FieldAdapters';
+import { AdditionalOptionsFormPartProps, PostgreSQLAdditionalOptionsProps } from '../FormParts.types';
+import { getStyles } from '../FormParts.styles';
+import { Messages } from '../FormParts.messages';
+import { trackingOptions } from '../FormParts.constants';
+
+export const AdditionalOptionsFormPart: FC<AdditionalOptionsFormPartProps> = ({
+  instanceType,
+  remoteInstanceCredentials,
+}) => {
+  const styles = getStyles();
+
+  return (
+    <div className={styles.groupWrapper}>
+      <h5>{Messages.form.titles.additionalOptions}</h5>
+      <div>
+        <CheckboxField
+          label={Messages.form.labels.additionalOptions.skipConnectionCheck}
+          name="skip_connection_check"
+        />
+        <CheckboxField label={Messages.form.labels.additionalOptions.tls} name="tls" />
+        <CheckboxField label={Messages.form.labels.additionalOptions.tlsSkipVerify} name="tls_skip_verify" />
+        {getAdditionalOptions(instanceType, remoteInstanceCredentials)}
+      </div>
+    </div>
+  );
+};
+export const PostgreSQLAdditionalOptions: FC<PostgreSQLAdditionalOptionsProps> = () => (
+  <Field
+    dataQa="tracking-options-radio-button-group"
+    name="tracking"
+    label={Messages.form.labels.trackingOptions}
+    options={trackingOptions}
+    component={RadioButtonGroupAdapter}
+  />
+);
+export const getAdditionalOptions = (type, remoteInstanceCredentials) => {
+  switch (type) {
+    case 'PostgreSQL':
+      return <PostgreSQLAdditionalOptions />;
+    case 'MySQL':
+      return (
+        <>
+          <CheckboxField
```

Comment:
Should this checkboxes here have data-qa? I think it may help QA

Reply:
these checkboxes have built it data-qa, no need to add it